src/Resources/app/administration/src/module/sw-in-app-purchases/component/sw-in-app-purchase-checkout-overview/sw-in-app-purchase-checkout-overview.html.twig

```diff
@@ -22,10 +30,39 @@
             {{ $t('sw-in-app-purchase-checkout-overview.excludeVat') }}
         </p>
 
+        <mt-checkbox
+            v-if="purchase.priceModels[0].conditionsType !== null"
+            :checked="gtcAccepted"
+            v-model:value="gtcAccepted"
+            class="sw-in-app-purchase-checkout-purchase__subtext__partner_gtc"
+            @update:checked="onGtcAcceptedChange"
+        >
+            <template #label>
+                {{ $t('sw-in-app-purchase-checkout-overview.partner.accept') }}
+                <mt-link
+                    :type="purchase.priceModels[0].conditionsType === 'gtc' ? 'external' : 'internal'"
+                    :as="purchase.priceModels[0].conditionsType === 'gtc' ? 'a' : 'button'"
+                    :href="purchase.priceModels[0].conditionsType === 'gtc' ? purchase.websiteGtc : undefined"
+                    @click="purchase.priceModels[0].conditionsType === 'detailed' ? openConditionsModal() : null"
+                >
+                    {{ $t('sw-in-app-purchase-checkout-overview.partner.gtc', { partner: producer }, 0) }}
+                </mt-link>
+            </template>
```

Comment:
Shouldn't the selected and not the first price model be used for these conditions?

Reply:
I spoke to Jannik about this when he first built it, and he stated that subscriptions wont have additional service conditions, just the standard shopware conditions. Where as services (be it `consumable` or `non-consumable`) will have additional service conditions. Since services can only have one `priceModel` per service, this will work.

I have a question to you, do you think I should put these conditions in the `index.ts` in `methods` and call it here (i.e. `getType(condition)`, `getAs(condition)` etc) or is this fine?

Reply:
Well, I would change it anyway. If the connditionsType is not part of the purchase but part of the price model, it indicates possible changes. To be more future proof, I would relay on the chosen pricing model rather than the first, just to be prepared for whatever comes next.

You can put it there as _computed_, but to be honest it's a lot of noise for something small. I would prefer a simple if-else and two mt-links there, it's much more convenient and readable.